src/slider/tests/functional/Slider.ts

```diff
@@ -0,0 +1,255 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import * as css from '../../styles/slider.m.css';
+import * as keys from 'leadfoot/keys';
+
+function getPage(test: any) {
+	const { browserName } = test.remote.environmentType;
+	if (browserName.toLowerCase() === 'microsoftedge') {
+		test.skip('example page currently doesn\'t work in edge.');
+	}
+	return test.remote
+		.get('http://localhost:9000/_build/common/example/?module=slider')
+		.setFindTimeout(5000);
+}
+
+function checkValue(command: any, values?: number[]) {
+	const notIE = command.session.capabilities.browserName.toLowerCase() !== 'internet explorer';
+	let currentValue: number;
+	return command
+		.findByCssSelector(`.${css.inputWrapper}`)
+			.findByCssSelector(`.${css.input}`)
+			.getProperty('value')
+			.then((value: string) => {
+				currentValue = parseInt(value, 10);
+			})
+			.end()
+			.findByCssSelector(`.${css.fill}`)
+			.getAttribute('style')
+			.then((style: string) => {
+				const absWidthRegex = /width:\s*(\d+)\.?\d*%/;
```

Comment:
I agree. I tried the property approach before coming up with this but it didn't work. I used `.getProperty('style')` but what got returned is an array of property names(`['width']`) in chrome, instead of an object with which I can then use `style.width`. If you point me to an example on how this can be done with property, I'd love to make the change.